packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesAsset.h

```diff
@@ -0,0 +1,26 @@
+//  Copyright © 2019 650 Industries. All rights reserved.
+
+@interface EXUpdatesAsset : NSObject
+
+/**
+ * properties determined by asset source
+ */
+@property (nonatomic, strong) NSURL * _Nonnull url;
+@property (nonatomic, strong) NSString * _Nonnull type;
+@property (nonatomic, strong) NSDictionary * _Nullable metadata;
+@property (nonatomic, strong) NSString * _Nullable nsBundleFilename; // used for embedded assets
```

Comment:
```suggestion
@property (nonatomic, strong) NSString * _Nullable mainBundleFilename; // used for embedded assets
```

(there can be multiple NSBundles, this makes it clearer)